docs/_pages/release-process.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Release process"
+permalink: /release-process/
+toc: false
+---
+
+The following steps must be executed to release a new version of Phoenicis:
+* Create release branches (e.g. "5.0-alpha") for phoenicis and scripts
+* Protect release branch
```

Comment:
I'm not sure I like it, that the release branch can be modified later on via pull requests.
The way I learned it, a release should be fixed, i.e. a frozen branch, which can't/shouldn't be modified.

When releasing a bugfix, or the next version later on, a new branch should be created.

Reply:
As you said: We use GitHub releases based on tags. So every tag is fixed. Therefore we can just release all releases which are based on the same master from the same branch instead of creating a new branch from the release branch. Apart from this, you see that we currently change the release branch because we detect bugs before actually triggering the release (the alternative to this would be to freeze master, which might stop development).

I'm open to change the way we do releases in the future but I think for now (alpha) we're ok.